js-data/js-data.d.ts

```diff
@@ -0,0 +1,404 @@
+// Type definitions for JSData v1.3.0
+// Project: https://github.com/js-data/js-data
+// Definitions by: Stefan Steinhart <https://github.com/reppners>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+///////////////////////////////////////////////////////////////////////////////
+// Promises in js-data are ES6 polyfill promises
+///////////////////////////////////////////////////////////////////////////////
+
+/// <reference path="../es6-promise/es6-promise.d.ts" />
+
+///////////////////////////////////////////////////////////////////////////////
+// js-data module (js-data.js)
+///////////////////////////////////////////////////////////////////////////////
+
+// defining what exists in JSData and how it looks
+declare module JSData_ {
```

Comment:
Nevermind - must have some other issue in the process. Now I was able to refactor it.